project-document/docs/database/database.md

```diff
@@ -50,7 +50,21 @@ Sort key is normally, explains the type of data, if the data is about User, the
 :::info
 
 This index is used for the access pattern of retrieving a list of users by the Group ID.
-The reason for not having Container and Group relation GSI, similar to User and Group relation, is that there are few chances to query groups by the container.
+
+:::
+
+### GroupAndContainerRelationship
+
+**GSI Name**: GroupAndContainerRelationship  
+**Projection Type**: KEY_ONLY
+
+| Key | Attribute |
+|:----|:----------|
+| PK  | ContainerId   |
+
+:::info
+
+This index is used for the access pattern of retrieving a GroupId by the Container ID.
```

Comment:
It's a minute thing and my preference, but unifying ID is better. You wrote it as the different patterns in the same sentence. (`GroupId` and `Container ID`)

Reply:
Thank you for mentioning it! I fixed it